x-pack/plugins/screenshotting/server/browsers/chromium/driver.ts

```diff
@@ -282,8 +285,8 @@ export class HeadlessChromiumDriver {
       captureBeyondViewport: false, // workaround for an internal resize. See: https://github.com/puppeteer/puppeteer/issues/7043
     });
 
-    if (Buffer.isBuffer(screenshot)) {
-      return screenshot;
+    if (screenshot.byteLength) {
+      return Buffer.from(screenshot);
```

Comment:
Is this just optimization or did some API change?

Reply:
There's been a change in the API from puppeteer... screenshots now return TypedArrays in place of buffers, see https://github.com/puppeteer/puppeteer/releases/tag/puppeteer-core-v23.0.0

Reply:
Thanks, I see mention of this in the 23.0.0 release notes as well :)